src/core_io.h

```diff
@@ -31,6 +31,6 @@ UniValue ValueFromAmount(const CAmount& amount);
 std::string FormatScript(const CScript& script);
 std::string EncodeHexTx(const CTransaction& tx, const int serializeFlags = 0);
 void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool fIncludeHex);
-void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry);
```

Comment:
Don't care about the order, though I'd prefer not using default arguments. Easy enough to just add these on all call sites, there aren't many.